src/xrpld/rpc/detail/RPCHelpers.cpp

```diff
@@ -385,20 +385,23 @@ ledgerFromRequest(T& ledger, JsonContext& context)
         return getLedger(ledger, ledgerHash, context);
     }
 
-    auto const index = indexValue.asString();
+    if (indexValue.isConvertibleTo(Json::stringValue))
```

Comment:
Do we support both forms  `index: 3` and` index: "3"` ?

Reply:
More accurately, I think we should.